modules/core/src/main/scala/org/tessellation/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,35 @@
+package org.tessellation.domain.trust.storage
+
+import cats.Order
+import cats.syntax.contravariant._
+
+import org.tessellation.cli.method.snapshotInterval
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.circe.magnolia.encoder
+import derevo.derive
+
+@derive(encoder)
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
+)
+
+object OrdinalTrustMap {
+
+  implicit val order: Order[OrdinalTrustMap] = Order[SnapshotOrdinal].contramap(_.ordinal)
+
+  implicit def catsOrder(implicit ord: Ordering[OrdinalTrustMap]): Order[OrdinalTrustMap] = ord.compare(_, _)
```

Comment:
why you need `Order[OrdinalTrustMap]` specified twice? the first order should be sufficient. also, it looks like ambiguous implicits

Reply:
`catsOrder` has been removed.